tests/models/encoder_decoders/wikitables_semantic_parser_test.py

```diff
@@ -25,6 +24,82 @@ def setUp(self):
     def test_model_can_train_save_and_load(self):
         self.ensure_model_can_train_save_and_load(self.param_file)
 
+    def test_correct_neighbor_indexes(self):
```

Comment:
I think this test would be more informative if you constructed a small world with just a few entities, and actually checked the whole returned tensor, instead of just a couple of entries in the tensor.  This would make sure that padding is correct (especially if we're padding to `num_neighbors` instead of `num_entities`), and that everything is as we expect.

Reply:
Also, there isn't a single "right" way to name a test, but what we've generally done is this: `test_[method_name]_[aspect_under_test]`.  See, for example, the tests here: https://github.com/allenai/allennlp/blob/wikitables/tests/data/semparse/knowledge_graphs/table_knowledge_graph_test.py.  This makes it a lot easier to know what the point of the test is, and if you end up breaking the test later, you can see in the name what functionality you've broken.

There are also times when there isn't a clear "aspect" that's under test, other than the standard flow through the method with a typical expected input.  In those cases, I typically just name it `test_[method_name]`.  I'd say this test falls under that category, so I'd name this `test_get_neighbor_indices`.

The same goes for the other test names here.